keanu-project/src/main/java/io/improbable/keanu/network/BayesianNetwork.java

```diff
@@ -63,6 +81,22 @@ public Vertex getVertexByLabel(VertexLabel label) {
         return vertices;
     }
 
+    public int getVertexCount() {
+        return vertexCount;
```

Comment:
You can just `return this.vertices.size();` and there's no need to store `vertexCount` as a field